### PR TITLE
Link BC backend to BC rest endpoints start-process and analyse-next-level

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

**Comment by janinko** [comment]
retest this please

File: bc-rest/src/main/java/org/jboss/da/bc/BuildConfigurationsProduct.java [hunk 1]
```diff
@@ -20,11 +19,26 @@
 import com.wordnik.swagger.annotations.ApiParam;
 import com.wordnik.swagger.annotations.ApiResponse;
 import com.wordnik.swagger.annotations.ApiResponses;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import org.apache.maven.scm.ScmException;
```
Comment (thescouser89) [inline bc-rest/src/main/java/org/jboss/da/bc/BuildConfigurationsProduct.java]: Doesn't respect the import order :cry:

File: communication/src/main/java/org/jboss/da/communication/pom/PomReader.java [hunk 2]
```diff
@@ -8,8 +8,16 @@
...
 import javax.xml.bind.Unmarshaller;
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
 import java.io.InputStream;
 import java.util.Optional;
+import javax.xml.transform.Source;
+import javax.xml.transform.sax.SAXSource;
+import org.jboss.da.communication.pom.impl.NamespaceFilter;
+import org.xml.sax.InputSource;
```
Comment (thescouser89) [inline communication/src/main/java/org/jboss/da/communication/pom/PomReader.java]: this does not respect the import order.

File: bc-rest/src/main/java/org/jboss/da/bc/model/BuildConfiguration.java [hunk 3]
```diff
@@ -9,6 +9,7 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 import lombok.ToString;
+import org.jboss.da.communication.model.GAV;
```
Comment (thescouser89) [inline bc-rest/src/main/java/org/jboss/da/bc/model/BuildConfiguration.java]: this does not respect the import order.

File: communication/src/main/java/org/jboss/da/communication/pom/model/MavenProject.java [hunk 4]
```diff
@@ -3,14 +3,16 @@
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlElement;
-import javax.xml.bind.annotation.XmlRootElement;
 import java.util.List;
+import javax.xml.bind.annotation.XmlRootElement;
```
Comment (thescouser89) [inline communication/src/main/java/org/jboss/da/communication/pom/model/MavenProject.java]: this does not respect the import order.

**Comment by jbartece** [comment]
Fix the import order next time please, but I'll  merge it not to block next work
:+1:

**Pull request merged**